source/index.html.md

```diff
@@ -299,6 +299,9 @@ Example response (snippet):
 }
 ````
 
+Depending on card scheme and merchant category, name on card might be necessary
```

Comment:
```suggestion
Depending on the card scheme and the merchant category, the name on the card might be necessary
```

To be honest, I don't know if this is a general sentiment to the text (consistency may win then), but isn't it slightly danglish to leave out "the" in front of things here, or is it because it's considered "properties" rather than the actual words? I'd personally make it obvious by quotation, e.g.: depending on "card scheme" and "merchant category", "name on card" might be..

Reply:
I had almost all the suggested "the"s initially, but then I thought it was "the" all over the place :joy: 

I tend to agree with your view.

@ct-clearhaus Any input on this?

Reply:
Right, I personally favor the quotations over a missing "the".
Sometimes it's also liberating to read a technical text that uses plain English which arguably require more thought to get correct as you can't use tech or industry made-up lingo.

I think, I went too far with the quotations in my example, e.g. "card scheme" probably doesn't refer to "a thing" you've defined in that past, so that should probably have "the" in front.

Reply:
🤷‍♂